flujo/state/granular_blob_store.py

```
@@ -0,0 +1,308 @@
+
+        Returns:
+            Deserialized payload
+
+        Raises:
+            BlobNotFoundError: If blob cannot be found
+        """
+        storage_key = f"{self.BLOB_NAMESPACE}:{blob_ref.blob_id}"
+
+        try:
+            blob_data = await self._backend.load_state(storage_key)
+        except Exception:
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)